statistics/histogram.go

```diff
@@ -1289,6 +1285,7 @@ func (idx *Index) expBackoffEstimation(sc *stmtctx.StatementContext, coll *HistC
 	//   2. Sort them and choose the first 4 most selective filter and the corresponding selectivity is sel_1, sel_2, sel_3, sel_4 where i < j => sel_i < sel_j.
 	//   3. The final selectivity would be sel_1 * sel_2^{1/2} * sel_3^{1/4} * sel_4^{1/8}.
 	// This calculation reduced the independence assumption and can work well better than it.
+	logutil.BgLogger().Warn("exp backoff", zap.String("range", fmt.Sprintf("%v", indexRange)), zap.String("col ids", fmt.Sprintf("%v", colsIDs)))
```

Comment:
Should we print column names here as well?

Reply:
Oh it's debug will be removed.